docs/tutorials/podman-on-windows.md

```diff
@@ -0,0 +1,331 @@
+Podman for Windows
+==================
+
+While "containers are Linux," Podman also runs natively on Mac and Windows, where it embeds a guest Linux system to launch your containers. This guest is referred to as a Podman machine and is managed with the podman machine command. On Windows, each machine is backed by a virtualized Windows System for Linux (WSLv2) distribution. The podman command can be run directly from your Windows PowerShell (or CMD) prompt, where it remotely communicates with the podman service running in the WSL environment. Alternatively, you can access Podman directly from the WSL instance if you prefer a Linux prompt and Linux tooling. In addition to command-line access, Podman also listens for Docker API clients, supporting direct usage of Docker-based tools and programmatic access from your language of choice.
```

Comment:
Nit just small question here since I am not sure: here in `runs natively on Mac and Windows` is `native` correct word cause in `macOS` podman instance runs inside a VM and on windows it uses `WSL` which is again compatibility layer and runs `linux kernel` on lightweight utility virtual machine.

Reply:
@n1hility PTAL

Reply:
Sure I'll try to make this clearer